advanced-pos-system/src/app/reports/page.tsx

```
@@ -0,0 +1,282 @@
+  if (!dateFrom && !dateTo && selectedRange !== 'all') {
+    const today = new Date();
+    const tomorrow = new Date(today);
+    tomorrow.setDate(today.getDate() + 1);
+    dateTo = getISODateString(tomorrow); // Supabase to is exclusive, so use start of next day
+
+    switch (selectedRange) {
+      case 'today':
+        dateFrom = getISODateString(today);
+        break;
+      case '7d':
+        const sevenDaysAgo = new Date();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)